frontend/src/app/components/transaction/transaction.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="container-xl">
 
   <div class="title-block">
-    <div *ngIf="rbfTransaction" class="alert alert-mempool" role="alert">
+    <div *ngIf="rbfTransaction && !tx.status.confirmed" class="alert alert-mempool" role="alert">
```

Comment:
Seems like this require some tx? null check

Reply:
yep, I just pushed a fix